docs/sources/mimir/configure/configure-spread-minimizing-tokens/index.md

```diff
@@ -103,6 +108,16 @@ If everything went smoothly, you should see something like this:
 
 Repeat steps 1 to 4, replacing all the occurrences of `zone-a` with `zone-b`.
 
-## Step 6: migrate ingesters from `zone-c`
+## Step 6: Migrate ingesters from `zone-c`
 
 Repeat steps 1 to 4, replacing all the occurrences of `zone-a` with `zone-c`.
+
+## Step 7: Delete the old token files
+
+If before the migration ingesters were configured to store their tokens under `-ingester.ring.tokens-file-path`, these files must be deleted once all ingester zones are migrated to the spread-minimizing tokens.
+
+For example, assuming that an ingester pod `ingester-zone-a` from a namespace `mimir-prod` used to store its tokens in a file `/data/tokens`, the latter can be deleted by executing:
+
```

Comment:
We might want to rephrase this step to avoid passive tense. For example:

If, before the migration, you configured ingesters to store their tokens under `-ingester.ring.tokens-file-path`, you must delete these files after migrating all ingester zones to spread-minimizing tokens.

For example, if an ingester pod called `ingester-zone-a` from a namespace called `mimir-prod` used to store its tokens in a file called `/data/tokens`, you can run the following command to delete the `/data/tokens` file:

Reply:
Thank you